src/Image/Image.tsx

```diff
@@ -17,12 +17,33 @@ export interface ImageProps {
 
 export const Image: React.FunctionComponent<ImageProps> = React.memo(
     (props: ImageProps): React.ReactElement<void> => {
+        const [imageClassName, setImageClassName] = React.useState<string>("");
+
+        React.useEffect(() => {
+            let className: string = "";
+            if (!props?.useImgTag) {
+                className += "div-tag";
+            } else {
+                className += "img-tag";
+            }
```

Comment:
this can be simplified to `let className: string = props?.useImgTag ? "img-tag" : "div-tag";`

or reverse the if else condition so you don't need `!` in the first if condition

Reply:
LoL its the same with the render/UI. Kind of lazy i guess

Reply:
extra `!` do extra unnecessary one bit work, haha, please change